src/clr-angular/popover/tooltip/providers/tooltip-id.service.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Subject } from 'rxjs';
```

Comment:
😱 breaching the DRY principle :trollface: 

nit: we could import them from the same `rxjs`?

Reply:
Sure thanks for pointing it out, I'll combine them into one. IntelliJ handles this and keeps them hidden in collapsed comments.